pkg/install/install.go

```diff
@@ -61,8 +61,11 @@ func RunYaml(cmd *cobra.Command, args []string) {
 // CmdUpgrade returns a CLI command
 func CmdUpgrade() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "upgrade",
+		Use:   "upgrade --noobaa-image <> --operator-image <>",
```

Comment:
Since in other commands, it is with a short flag description inside the pointy brackets `<>` (for example: in backingstore).
https://github.com/noobaa/noobaa-operator/blob/c0fc2d445a4dbaf7806f18d02f493d8dd8f8519a/pkg/backingstore/backingstore.go#L56

```suggestion
		Use:   "upgrade --noobaa-image <noobaa-image-version> --operator-image <operator-image-version>",
```